### ISSUE TITLE
Add tab icons for GNUSocial and Mastodon

### ISSUE THREAD

**Issue by rugk** [TwidereProject/Twidere-Android#1098]
When choosing the icons for the tab pages, one can choose from several icons, including the Twitter icon.

As Twidere also supports GNUSocial and Mastodon, it would be nice, if you could add these icons, so one can also choose these.
This is especially useful, as one then has a nice icon for a "only-GNUSocial/Mastodon" timeline, which you can configure there.

**Comment by Tedko** [comment]
Nice idea! @rugk Just let you know Twidere is back online and maintained by community, supported by https://dimension.im . Pls spread the word and let ppl know! We would like to bring as many community contributor as possible. Many thanks.

**Comment by rugk** [comment]
Hmm, will there be a new release? [The last one was in Februrary](https://github.com/TwidereProject/Twidere-Android/releases/tag/4.0.3).

**Comment by Tedko** [comment]
> Hmm, will there be a new release? [The last one was in Februrary](https://github.com/TwidereProject/Twidere-Android/releases/tag/4.0.3).

Yes - working on one - will be released very soon - gonna fix Twitter DM first
